grml-debootstrap

```diff
@@ -308,14 +310,19 @@ cleanup() {
   fi
 
   if [ -n "${ORIG_TARGET}" ] ; then
-    einfo "Removing loopback mount of file ${ORIG_TARGET}."
-    kpartx -d "${ORIG_TARGET}"
-    # Workaround for a bug in kpartx which doesn't clean up properly,
-    # see Debian Bug #891077 and Github-PR grml/grml-debootstrap#112
-    if dmsetup ls | grep -q "^${LOOP_PART} "; then
-      kpartx -d "/dev/${LOOP_DISK}" >/dev/null
+    if [ "$VMTYPE" = "raw" ]; then
+      einfo "Removing loopback mount of file ${ORIG_TARGET}."
+      kpartx -d "${ORIG_TARGET}"
+      # Workaround for a bug in kpartx which doesn't clean up properly,
+      # see Debian Bug #891077 and Github-PR grml/grml-debootstrap#112
+      if dmsetup ls | grep -q "^${LOOP_PART} "; then
+        kpartx -d "/dev/${LOOP_DISK}" >/dev/null
+      fi
+      eend $?
+    else
+      einfo "Removing loopback mount of file ${ORIG_TARGET}."
+      qemu-nbd -d /dev/nbd0
     fi
-    eend $?
   fi
```

Comment:
@paulmenzel Just a quick question: you moved `eend $?`. Any reason for not keeping it at the end?

Reply:
Thank you. That’s an oversight no my part.